Code_Exercises/9_sycl_of_hell/Makefile

```diff
@@ -0,0 +1,22 @@
+CXX ?= icpx
```

Comment:
Why do we need a `Makefile` as well as a `CMakeLists.txt`?

Reply:
Because I hate CMakeList and have no clue how do debug it.
More seriously because the CMakeList will not work for adaptive CPP and like the simplificy of Makefile.